### PR TITLE
fix: refactor contact add

### PR DESCRIPTION
**Pull request by limpbrains**
### Description

- upgrade synonymdev/slashtags-url
- refactor contact add form so that user can type slash url + relay

### Type of change

Bug fix

### Tests

Detox test

### CODE REVIEW

**Comment by pwltr** [comment]
Couple of findings:
- In the design the text has the brand color on error state 

image

- The field is overlaying the button when a valid url is passed

Simulator Screenshot - iPhone 15 - 2023-11-28 at 18 19 59

- There is a slight layout shift when the error message appears
- I think the paste button shouldn't require pressing 'Add' again if the url was valid
- The position for the error text is a bit suboptimal when it comes to multi-lang support. Quickly tested german text and it wouldn't fit in one row together with the label on the left.

**Comment by limpbrains** [comment]
> In the design the text has the brand color on error state
>  The field is overlaying the button when a valid url is passed
> There is a slight layout shift when the error message appears
> I think the paste button shouldn't require pressing 'Add' again if the url was valid

fixed

> The position for the error text is a bit suboptimal when it comes to multi-lang support. Quickly tested german text and it wouldn't fit in one row together with the label on the left.

Agree. I've added text wrap. Now if error message doesn't fit, it will appear on the new line

**Review by pwltr — APPROVED** [review]

File: src/screens/Contacts/AddContact.tsx [hunk 1]
```diff
@@ -27,57 +28,64 @@ const AddContact = ({
...
-		setError(false);
+	const handleChangeUrl = (contactUrl: string): void => {
+		setUrl(contactUrl);
+		setError(undefined);
+	};
 
-		if (url === '') {
+	const handleAddContact = (contactUrl?: string): void => {
+		contactUrl = contactUrl ?? url;
+		console.info('handleAddContact', contactUrl);
```
Comment (pwltr) [inline src/screens/Contacts/AddContact.tsx]: console.log

Comment (limpbrains) [reply src/screens/Contacts/AddContact.tsx]: oops

**Pull request merged**